src/cmd/linuxkit/run_qemu.go

```diff
@@ -452,8 +453,13 @@ func runQemuContainer(config QemuConfig) error {
 		}
 	}
 
+	currentUser, err := user.Current()
+	if err != nil {
+		return err
+	}
+	userString := fmt.Sprintf("%s:%s", currentUser.Uid, currentUser.Gid)
 	dockerArgs := append([]string{"run", "--interactive", "--rm", "-w", cwd}, binds...)
-	dockerArgsImg := append([]string{"run", "--rm", "-w", cwd}, binds...)
+	dockerArgsImg := append([]string{"run", "--user", userString, "--rm", "-w", cwd}, binds...)
```

Comment:
I assume this is necessary when running on GitHub actions. Are there any knockdown effects? This does get called anytime someone calls `lkt run ` with qemu, does it not?

Reply:
It's not just for actions (see commit message for more info).
TL;DR on Linux a disk created using containerized `qemu-img` will be owned by `root:root`.

Adding `sudo` to a test seemed like a bad idea and this was the best alternative.

If someone with a mac (without qemu installed) could run this from  the `tests` directory it should confirm whether or not this breaks anything.
```
rtf -vvv -l build run -x linuxkit.package.extend.ext4
```

Reply:
I would do it, but definitely have `qemu` installed. It will not help.

Reply:
I think the containerized qemu is a mistake, and we should remove support for it.

Reply:
@justincormack I'll take a look at removing it + then we can rebase this PR on top of that.